### PR TITLE
add menhir skeleton

### PR DESCRIPTION
**Pull request by zapashcanon**

### CODE REVIEW

File: drom.opam [hunk 1]
```diff
@@ -26,7 +26,7 @@ bug-reports: "https://github.com/ocamlpro/drom/issues"
 dev-repo: "git+https://github.com/ocamlpro/drom.git"
 tags: "org:ocamlpro"
 build: [
-  ["dune" "subst"] {dev}
```
Comment (lefessan) [inline drom.opam:29]: In this comment https://github.com/ocaml/opam-repository/pull/17730#issuecomment-735456736 , Kate asked us to switch from `{pinned}` to `{dev}`

File: share/drom/skeletons/packages/menhir_lib/dune_ [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  !{dune-stanzas}
+  !(package-dune-stanzas)
+  )
+
+!{package-dune-files}
+(documentation
+  (package !{package-name}))
+!(package-dune-trailer)
+
+(ocamllex
```
Comment (lefessan) [inline share/drom/skeletons/packages/menhir_lib/dune_:16]: These lines are automatically added by `drom` if the line `generators = [ "ocamllex", "menhir" ]` is in the `package.toml` file.

**Comment by lefessan** [comment]
If you rebase on the last `master` (10 minutes ago...), you should be able to write `menhir_lib/package.toml` file for the package skeleton, in particular, it should contain a dependency towards `menhirLib`, and a line `generators = [ "ocamllex", "menhir" ]`, no ?

I have used the convention to use dashes instead of underscores in skeleton names until now. Either we switch all of them to underscores, or this skeleton should follow the same convention.

File: drom.opam (hunk 1 above)
Comment (zapashcanon) [reply drom.opam:29]: This file is generated, I didn't touched it by hand, so I'm not sure why it changed.

File: share/drom/skeletons/packages/menhir/dune_ [hunk 3]
```diff
@@ -0,0 +1,21 @@
+![file:dune]; generated by drom from package skeleton 'library'
```
[the remark this answers is not in the document]
Comment (zapashcanon) [reply share/drom/skeletons/packages/menhir/dune_]: OK, I copy-pasted that from some other skeleton, I'll fix all of them then.

File: share/drom/skeletons/packages/menhir_lib/dune_ (hunk 2 above)
Comment (zapashcanon) [reply share/drom/skeletons/packages/menhir_lib/dune_]: I added a `package.toml` file with the corresponding line and it doesn't seems to work.

**Pull request merged**